build-wine64.txt

```diff
@@ -5,6 +5,7 @@ ar = 'ar'
 strip = 'strip'
 
 [properties]
+needs_exe_wrapper = true
```

Comment:
Why does this need to be set when we actually *don't* want wine to execute?

Reply:
If I correctly understand meson documentation -- this will stop meson from guessing/using wrapper:

> One important thing to note, if you did not define an exe_wrapper in the previous section, is that Meson will make a best-effort guess at whether it can run the generated binaries on the build machine. It determines whether this is possible by looking at the system and cpu_family of build vs host. There will however be cases where they do match up, but the build machine is actually not compatible with the host machine. Typically this will happen if the libc used by the build and host machines are incompatible, or the code relies on kernel features not available on the build machine. One concrete example is a macOS build machine producing binaries for an iOS Simulator x86-64 host. They're both darwin and the same architecture, but their binaries are not actually compatible. In such cases you may use the needs_exe_wrapper property to override the auto-detection:
```
[properties]
needs_exe_wrapper = true

```